notebooks/official/prediction/get_started_with_tf_serving.ipynb

```diff
@@ -700,7 +666,7 @@
         "\n",
         "Both the preprocessing and post-processing functions are converted to static graphs which are fused to the model. The output from the underlying model is passed to the post-processing function. The post-processing function passes the converted/packaged output back to the HTTP server. The HTTP server returns the output as the HTTP response content.\n",
         "\n",
-        "One consideration you need to consider when building serving functions for TF.Keras models is that they run as static graphs. That means, you cannot use TF graph operations that require a dynamic graph. If you do, you will get an error during the compile of the serving function which will indicate that you are using an EagerTensor which is not supported."
+        "One consideration you need to consider when building serving functions for TF.Keras models is that they run as static graphs. That means, you cannot use TF graph operations that require a dynamic graph. If you do, you get an error during the compile of the serving function which indicates that you are using an EagerTensor which is not supported."
```

Comment:
1. change "cannot" to  "can't" (i.e., contraction)
2. change "you are" to "you're"
3.  change "is not" to "isn't"